### ISSUE TITLE
Failure to build 0.10.0 with simd feature enabled

### ISSUE THREAD

**Issue by bbarker** [tazz4843/whisper-rs#105]
```
   Compiling whisper-rs v0.10.0
error[E0432]: unresolved import `std::simd::SimdInt`
 --> /home/bbarker/.cargo/registry/src/index.crates.io-6f17d22bba15001f/whisper-rs-0.10.0/src/utilities.rs:2:33
  |
2 | use std::simd::{f32x16, i16x16, SimdInt};
  |                                 ^^^^^^^ no `SimdInt` in `simd`
  |
  = help: consider importing one of these items instead:
          core::simd::prelude::SimdInt
          std::simd::prelude::SimdInt

error[E0599]: no method named `cast` found for struct `Simd` in the current scope
  --> /home/bbarker/.cargo/registry/src/index.crates.io-6f17d22bba15001f/whisper-rs-0.10.0/src/utilities.rs:40:46
   |
40 |         let simd = i16x16::from_slice(chunk).cast::<f32>();
   |                                              ^^^^ method not found in `Simd<i16, 16>`
  --> /rustc/eeff92ad32c2627876112ccfe812e19d38494087/library/core/src/../../portable-simd/crates/core_simd/src/simd/num/int.rs:26:8
   |
   = note: the method is available for `Simd<i16, 16>` here
   |
   = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
   |
1  + use std::simd::num::SimdInt;
   |

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `whisper-rs` (lib) due to 2 previous errors

```

**Comment by bbarker** [comment]
```
$rustc --version
rustc 1.76.0-nightly (eeff92ad3 2023-12-13)

```

**Comment by tazz4843** [comment]
Doing some benchmarking, it seems the SIMD functions aren't any faster. It's likely easier to just remove them entirely.

**Comment by tazz4843** [comment]
Closed in 249b7321c99e2578c47364c9982ff5b0e57f3605
